numba/cuda/mathimpl.py

```diff
@@ -76,8 +88,20 @@ def lower_boolean_impl(context, builder, sig, args):
 
 def impl_unary(key, ty, libfunc):
     def lower_unary_impl(context, builder, sig, args):
-        libfunc_impl = context.get_function(libfunc, typing.signature(ty, ty))
-        return libfunc_impl(builder, args)
+        if ty == float32 and context.fastmath is True:
```

Comment:
```suggestion
        if ty == float32 and context.fastmath:
```

Reply:
@stuartarchibald thanks for the review, I have made the code changes locally. Can you point me to the documentation files that need to be changed?